cubedash/logs.py

```diff
@@ -6,7 +6,7 @@
 from typing import BinaryIO
 
 import structlog
-from orjson import orjson
+from orjson.orjson import OPT_SORT_KEYS, dumps
```

Comment:
The orjson docs use `import orjson` and access functions and options from there. I think accessing everything directly on the `orjson` module is clearer. `dumps` is a very generic function name on it's own.

Example from the docs.
```python
import orjson, datetime, numpy
data = {
    "type": "job",
    "created_at": datetime.datetime(1970, 1, 1),
    "status": "🆗",
    "payload": numpy.array([[1, 2], [3, 4]]),
}
orjson.dumps(data, option=orjson.OPT_NAIVE_UTC | orjson.OPT_SERIALIZE_NUMPY)
```

Reply:
It's possible there is something with the environment in Explorer, but both MyPy and Pyright disagrees with the documentation when I run `uv run <typechecker> cubedash/logs.py`.